src/Microsoft.IdentityModel.Protocols.OpenIdConnect/Configuration/OpenIdConnectConfiguration.cs

```diff
@@ -93,6 +94,27 @@ public static string Write(OpenIdConnectConfiguration configuration)
             return OpenIdConnectConfigurationSerializer.Write(configuration);
         }
 
+        /// <summary>
+        /// Writes an <see cref="OpenIdConnectConfiguration"/> to a json string using the provided <see cref="Stream"/>.
```

Comment:
This wording confuses me.

/// Writes an <see cref="OpenIdConnectConfiguration"/> as JSON to the <paramref "name"="stream">

maybe?